vmdb/app/models/miq_request.rb

```diff
@@ -226,8 +197,7 @@ def get_options
   end
 
   def request_type_display
-    return "Unknown" if self.request_type.nil?
-    REQUEST_TYPES[self.type.to_sym][self.request_type.to_sym].to_s
+    request_types.fetch_path(type.to_sym, request_type.try(:to_sym)).to_s || "Unknown"
```

Comment:
I think you can just completely remove the .to_s.  If it's missing it will be `nil || "Unkown"`  If it's not, then it's always a string, right?